Content.Server/Corvax/TTS/TTSSystem.Sanitize.cs

```diff
@@ -56,7 +56,11 @@ private string ReplaceWord2Num(Match word)
             {"гсб", "Гэ Эс Бэ"},
             {"гв", "Гэ Вэ"},
             {"нр", "Эн Эр"},
-            {"км", "Кэ Эм"},
+            {"нра", "Эн Эр А"},
+            {"нру", "Эн Эр У"},
```

Comment:
Мне кажется, так будет правильнее
```suggestion
            {"нра", "Эн Эра"},
            {"нру", "Эн Эру"},
```